cigo_wrapper/network/CigoConnect.py

```diff
@@ -179,5 +182,35 @@ def retrieve_a_itinerary(self, itinerary_id):
 
         self.__raise_response_exception(response)
 
+    def retrieve_job_latest_geolocation(self, job_id):
+        """
+        Retrieve the approximate geolocation of the assigned Operator.
+        
+        The response also contains their last known ETA and distance from 
+        the Job's geolocation.
+        """
+        url = self.base_url.format(f'jobs/id/{job_id}/location')
+        response = requests.get(url, auth=(self.account_id, self.auth_key))
+        data = response.json()
+
+        if data['statusCode'] != 200:
+            return self.__raise_response_exception(response)
+
+        job_data = data["job"]
+        job = Job.from_json(job_data)
+
+        if job_data.get("progress"):
+            job.progress = JobProgress.from_json(job_data["progress"])
```

Comment:
This is not needed as it is assigned in job = Job.from_json(job_data)

Reply:
This is same as `coordinates`,  `geocoding` and `progress`.